crm_won_restrict_per_stage/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds possibility to set CRM stages that are being able to see won button and restrict change to an stage won that is not previous set to see won button
```

Comment:
On RST/MD files, it's interesting to cut at 88 cols the same as in code files.

This can also be rephrased for better understanding and ortography:

```suggestion
This module adds the possibility to set CRM stages that are being able to see the won
button (hiding it for the rest).

It also restricts to change to a won stage won from an stage that doesn't have this
check marked.
```

Reply:
There's also missing the file CONFIGURE.rst to explain the menu path where to change the check.